demisto_sdk/commands/common/hook_validations/description.py

```diff
@@ -137,3 +138,44 @@ def is_valid_description_name(self):
                 return False
 
         return True
+
+    def verify_demisto_in_description_content(self):
+        """
+        Checks if there are the word 'Demisto' in the description content.
+
+        Return:
+            True if 'Demisto' does not exist in the description content, and False if it does.
+        """
+
+        data_dictionary = get_yaml(self.file_path)
+        is_unified_integration = data_dictionary.get('script', {}).get('script', '') not in {'-', ''}
+        yml_line_num = 0
+
+        if is_unified_integration:
+            description_content = data_dictionary.get('detaileddescription', '')
+
+            with open(self.file_path, 'r') as f:
+
+                for line_n, line in enumerate(f.read().split('\n')):
+                    if 'detaileddescription:' in line:
+                        yml_line_num = line_n + 1
+        else:
+            try:
+                description_path = glob.glob(os.path.join(os.path.dirname(self.file_path), '*_description.md'))[0]
+
+            except IndexError:
+                error_message, error_code = Errors.no_description_file_warning()
+                if self.handle_error(error_message, error_code, file_path=self.file_path, warning=True):
+                    return True
+
+            with open(description_path) as f:
+                description_content = f.read()
+
+        for line_num, line in enumerate(description_content.split('\n')):
+            if 'demisto ' in line.lower() or ' demisto' in line.lower():
```

Comment:
let's gather all the lines which have the word in them and give one error back - not multiple times.

Reply:
done.